README.md

```diff
@@ -1,4 +1,54 @@
 # BSC-MEV-Sentry
 
-BSC-MEV-Sentry is a proxy service within the BSC MEV architecture designed to shield the validator network.
-It functions to forward requests and pay to builders.
+BSC-MEV-Sentry serves as the proxy service for BSC MEV architecture, It has the following features:
+
+1. Forward RPC requests under mev namespace to validators/builders.
+2. Pay builders for their bid.
+3. Monitor validators' status and health.
+
+See also: https://github.com/bnb-chain/BEPs/pull/322
+
+# Usage
```

Comment:
Can you explain the most important API `mev_params` and  each field inside it.

Right now, I can see the resposne only have: `{"ValidatorCommission":100,"BidSimulationLeftOver":50000000}`.

As far as I know, we have `gasCeil`, and `paybidLimit` atleast?

Reply:
is Sentry's readme suitable for explaining the details of mevparams.